.github/contributing.md

```diff
@@ -0,0 +1,62 @@
+# Contributing
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+## Pull Request Process
+
+These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to our repos or how we handle your PRs.
+
+1. Ensure any build dependencies or sensitive environment variables are removed before posting your PR.
+2. In the PR summary, note the features you're adding or changes to existing interfaces.
+3. Increase the version numbers ([example commit](https://github.com/Frameio/python-frameio-client/commit/3b6595aae9f8a0df984c937ca9938131afbb82c0)).  The versioning scheme we use is [SemVer](http://semver.org/).
```

Comment:
I think we should do automatic semver handling. Looks like we're using a tool to do this on massdriver, the webclient, and vapor.

Leaving it up to the users is a bit messy.

Reply:
I'll remove the request to increase version number and leave the note about SemVer as an FYI.